src/components/sdk/MeetingControls/VideoInputBackgroundBlurControl.tsx

```diff
@@ -54,6 +54,31 @@ const VideoInputBackgroundBlurControl: React.FC<Props> = ({
     };
   }, []);
 
+  async function toggleBackgroundBlur() {
```

Comment:
Why do we need to make this change? This function depends on `isVideoTransformChecked` so I thought the previous change makes more sense.

Reply:
There was an extra call to `selectVideoInputDevice` when we call this as part of useEffect. When you use a useEffect hook, the useEffect will be triggered when the component mounts, which is not the behaviour we wanted. We just want this to be a function triggered by on click event.

Reply:
Then we should pass the new value of `isVideoTransformChecked` inside the function here from the clickEvent.

Reply:
Ok - removed the ui state and depend on if the current selected device is a transformed device or not.